ui/app/src/components/FormsEngine/components/FormsEngineField.tsx

```
@@ -127,159 +131,170 @@ export const FormsEngineField = forwardRef<HTMLDivElement, FormsEngineFieldProps
-							>
-								View
-							</Button>
-							{/* TODO: Create or link to content inheritance article */}
+				{hasHelpText && (
+					<Collapse in={showHelp}>
+						<Alert severity="info" variant="outlined" sx={{ border: 'none' }}>
+							<Typography
+								variant="body2"
+								component="section"
+								color="textSecondary"
+								dangerouslySetInnerHTML={{ __html: field.helpText }}
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)